virtualizarr/tests/test_readers/conftest.py

```diff
@@ -222,7 +222,7 @@ def skip_test_for_libhdf5_version():
     return libhdf5_version < Version("1.14")
 
 
-@pytest.fixture(params=["blosc_zlib", ""])
+@pytest.fixture(params=["zlib", ""])
```

Comment:
This is because the netcdf4 wheels don't include plugins (xref https://github.com/Unidata/netcdf4-python/issues/1164). I'm not sure how important it is to use blosc specifically for this test.